pkgs/repo_query/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: repo_query
+description: Miscellaneous issue, repo, and PR query tools.
+
+publish_to: none
+
+environment:
+  sdk: ^2.19.0
+
+dependencies:
```

Comment:
Dude: there's pkg:github!

Reply:
I've not used it before; sounds like I should take a look (esp. if it can lop off ~100 lines of code here).

Reply:
Yeah, like don't hand-write JSON decoding logic!

Reply:
updated -